accounting/receivables/customer_payments/check.rst

```diff
@@ -1,3 +1,148 @@
-==================================
-How to record a payment by checks?
-==================================
+============================================
+How to register customer payments by checks
+============================================
+
+There are two ways to handle payments received by checks. Odoo support
+both approaches so that you can use the one that better fits your
+habits.
+
+1. Undeposited Funds: 
+    once you receive the check, you record a payment
```

Comment:
one space too much, should be:
```
1. Undeposited Funds: 
   once you receive the check, you record a payment
```
otherwise get
![screenshot from 2015-10-27 09-24-04](https://cloud.githubusercontent.com/assets/564822/10752749/7fdc186e-7c8c-11e5-94e7-3a5209d547e2.png)

and put "Undeposited Funds" in bold